source/gui/installerGui.py

```diff
@@ -96,7 +107,25 @@ def __init__(self, parent, isUpdate):
 			if not os.path.isdir(installer.defaultInstallPath):
 				# Translators: a message in the installer telling the user NVDA is now located in a different place.
 				msg+=" "+_("The installation path for NVDA has changed. it will now  be installed in {path}").format(path=installer.defaultInstallPath)
-		sHelper.addItem(wx.StaticText(self,label=msg))
+		if shouldAskAboutAddons:

```

Comment:
I would prefer having this button before continue in the tab order, so that a user will encounter this button before encountering the continue button and may be clicks it accidentally, not knowing that there's another one.

Reply:
Yes, I agree. This is actually inconsistent at the moment. I will fix this.